UnityPy/classes/Object.py

```diff
@@ -46,7 +46,12 @@ def dump_typetree_structure(self) -> str:
         return self.reader.dump_typetree_structure()
 
     def read_typetree(self, nodes: list = None) -> dict:
-        tree = self.reader.read_typetree(nodes)
+        tree = dict()
```

Comment:
This line is redundant.
``tree`` is assigned in both of the two branches, so there is no need to assign it beforehand.
Side note, using ``dict()`` is slower than using ``{}``, see [No, \[\] And list\(\) Are Different In Python](https://towardsdatascience.com/no-and-list-are-different-in-python-8940530168b0)